### ISSUE TITLE
Document pull request process

### ISSUE THREAD

**Issue by marefr** [grafana/grafana#17742]
Basically extract the current [contribution guideline](https://github.com/grafana/grafana/blob/master/CONTRIBUTING.md) to a new pull request document.

And add links from the contribution guideline to pull request guideline.
Add links to the [review process of pull reuqests](https://github.com/grafana/grafana/issues/17740).

Related/blocked by #17652
